runtime/src/accounts_db.rs

```diff
@@ -2279,33 +2373,33 @@ pub mod tests {
         accounts.store(current_slot, &[(&purged_pubkey2, &zero_lamport_account)]);
         accounts.add_root(current_slot);
 
-        current_slot += 1;
-        accounts.store(current_slot, &[(&dummy_pubkey, &dummy_account)]);
```

Comment:
This is now gone thanks to #8148

Reply:
Originally, this workaround is added here: https://github.com/solana-labs/solana/pull/8176/files#r376724066

To add next artificial slot, in addition of just `add_root`, I also had to store a dummy account otherwise missing bank hash error occurred.